presentation/src/main/java/com/indipage/ui/tutorial/TutorialFifthFragment.kt

```diff
@@ -0,0 +1,31 @@
+package com.indipage.ui.tutorial
+
+import android.content.Intent
+import android.os.Bundle
+import android.view.View
+import com.example.core_ui.base.BindingFragment
+import com.indipage.presentation.R
+import com.indipage.presentation.databinding.FragmentTutorialFifthBinding
+import com.indipage.ui.MainActivity
+
+class TutorialFifthFragment :
+    BindingFragment<FragmentTutorialFifthBinding>(R.layout.fragment_tutorial_fifth) {
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        initButton()
+    }
+
+    private fun initButton() {
+        binding.btnTutorialWelcome.setOnClickListener() {
+            val intent =
+                Intent(activity, MainActivity::class.java) //fragment라서 activity intent와는 다른 방식
+            intent.addFlags(Intent.FLAG_ACTIVITY_NO_ANIMATION)
+            startActivity(intent)
+            activity?.supportFragmentManager
+                ?.beginTransaction()
+                ?.remove(this)
+                ?.commit()
```

Comment:
```suggestion
            startActivity(Intent(requireActivity(), MainActivity::class.java))
            requireActivity().finish()
```

TutorialActivity -> MainActivity 큰 흐름이 엑티비티끼리 전환이니깐 이렇게 바꾸는건 어떤가유??

Reply:
이런 간단티비한 방법이..!! 덕분에 코드를 아주 많이 줄였습니다